src/components/structures/MatrixChat.tsx

```diff
@@ -394,7 +402,8 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         }
 
         const crossSigningIsSetUp = cli.getStoredCrossSigningForUser(cli.getUserId());
-        if (crossSigningIsSetUp) {
+        const crossSignInVerifySkip = SettingsStore.getValue(UIFeature.SkipPreloginSessionVerifyPrompt);
+        if (crossSigningIsSetUp && !crossSignInVerifySkip) {
```

Comment:
What is the functional impact of not allowing the user to verify their other devices?

Reply:
If you have not verified your session from other device then you may not be able to see encrypted messages. We have encryption switched off anyways so we do not need to verify device with cross sign in process. So, it fits with purpose.

Reply:
As discussed, if we can only safely not perform session verification if encryption is disabled on the home server, then remove this additional configuration parameter and instead look up if encryption is enabled in the well-known config of the home server.

Reply:
Looking up encryption parameter is failing as even though there is no reference to ch....local in config, the request is going back to chs which is odd. Only reference used as chs was at roomserver docker build. I wonder if that is changed to pegacorn... would be fix in future.

Reply:
Please refer to pull request #12